tests/test_metromap.py

```diff
@@ -0,0 +1,165 @@
+import io
+import unittest
+
+import pandas as pd
+
+try:
+    import networkx as nx
+except ImportError:
+    nx = None
+
+import gurobi_optimods.metromap as map
+
+# python -m unittest tests.test_metromap.Testmap
```

Comment:
Remove this line

Reply:
Can I leave it? It is also in the line optimization OptiMod, and it helps me when starting to work with them...

Reply:
Yeah actually it's quite nice to have a quick reference to the tests for the mod in an obvious place (since `make test` takes a long time now with all the mods we are accumulating ...). Maybe just add something to the effect of "to run the unit tests for this mod, use this command ..." so that it doesn't look so lonely.